src/plugins/workspace/public/components/workspace_menu/workspace_menu.tsx

```diff
@@ -42,21 +53,29 @@ function getFilteredWorkspaceList(
   return [
     ...(currentWorkspace ? [currentWorkspace] : []),
     ...workspaceList.filter((workspace) => workspace.id !== currentWorkspace?.id),
-  ].slice(0, 5);
+  ].slice(0, 7);
```

Comment:
Why 7? Do we want to use a constant? May want to update comment above.

Reply:
Let's make it a constant, e.g., `MAX_WORKSPACE_NUM`

Reply:
According the latest UX, I will update this constant to 5.